miso-web/src/main/webapp/scripts/lims.js

```diff
@@ -266,6 +266,16 @@ var Utils = Utils
               output[field.property] = input.value;
             };
             break;
+          case 'textarea':
+            input = document.createElement('TEXTAREA');
+            input.setAttribute('rows', field.rows || 10);
+            input.setAttribute('cols', field.cols || 50);
```

Comment:
The search text area is wider than the dialog containing it. Please make it smaller so it doesn't create a scroll bar.